test/metabase/upload_test.clj

```diff
@@ -1262,18 +1262,46 @@
-                  (let [csv-rows ["biginteger,float,text,boolean,date,datetime,offset_datetime"
-                                  "2000000,2.0,some_text,true,2020-02-02,2020-02-02T02:02:02,2020-02-02T02:02:02+02:00"]
+                                               :datetime        datetime-type))
+                           :rows [[1000000,1.0,"some_text",false,#t "2020-01-01",#t "2020-01-01T00:00:00"]]})]
+                  (let [csv-rows ["biginteger,float,text,boolean,date,datetime"
+                                  "2000000,2.0,some_text,true,2020-02-02,2020-02-02T02:02:02"]
                         file  (csv-file-with csv-rows)]
                     (is (some? (update-csv! action {:file file, :table-id (:id table)})))
                     (testing "Check the data was uploaded into the table correctly"
                       (is (= (set (updated-contents
                                    action
-                                   [[1000000 1.0 "some_text" false "2020-01-01T00:00:00Z" "2020-01-01T00:00:00Z" "2020-01-01T00:00:00Z"]]
-                                   [[2000000 2.0 "some_text" true "2020-02-02T00:00:00Z" "2020-02-02T02:02:02Z" "2020-02-02T00:02:02Z"]]))
+                                   [[1000000 1.0 "some_text" false "2020-01-01T00:00:00Z" "2020-01-01T00:00:00Z"]]
+                                   [[2000000 2.0 "some_text" true "2020-02-02T00:00:00Z" "2020-02-02T02:02:02Z"]]))
+                             (set (rows-for-table table)))))
+                    (io/delete-file file)))))))))))
+
+(deftest update-offset-datetime-test
+  (mt/test-drivers (mt/normal-drivers-with-feature :uploads)
+    (doseq [action (actions-to-test driver/*driver*)]
+      (testing (action-testing-str action)
+        (with-mysql-local-infile-on-and-off
+          (mt/with-report-timezone-id! "UTC"
+            (testing "Append should succeed for offset datetime columns"
+              (with-redefs [driver/db-default-timezone (constantly "Z")
+                            upload/current-database    (constantly (mt/db))]
+                (with-upload-table!
+                  [table (create-upload-table!
+                          {:col->upload-type (columns-with-auto-pk
+                                              (ordered-map/ordered-map :offset_datetime offset-dt-type))
+                           :rows []})]
+                  (let [csv-rows ["offset_datetime"
+                                  "2020-02-02T02:02:02+02:00"]
+                        file  (csv-file-with csv-rows (mt/random-name))]
+                    (is (some? (update-csv! action {:file file, :table-id (:id table)})))
+                    (testing "Check the data was uploaded into the table correctly"
+                      (is (= (set (updated-contents
+                                   action
+                                   ;; TODO
```

Comment:
Can you add more color to this comment and perhaps an issue link?

Reply:
Oops nevermind, see I jumped the gun on reviewing.

Reply:
Oops again, see you did request a review and I tripped over the way GH implicitly unassigns the team after I comment.

Reply:
Oops, that wasn't meant to be left there. Removed